About.html

```diff
@@ -1,56 +1,30 @@
 <!DOCTYPE html>
 <html>
     <link href="Project.css" rel="stylesheet">
-    <script>

```

Comment:
the js didn't work properly so removed that part and instead have a standard (html and css only) navbar now